wp-cli/README.md

```diff
@@ -1,88 +1,102 @@
 
-#### wp civicrm process-mail-queue
+There is a new command `wp civicrm db <command>` which (sort of) mirrors the functionality in `wp db <command>`. It holds the commands that apply to interaction with the CiviCRM database.
 
-Process pending CiviMail mailing jobs.
+| Command | Description | Old Command |
+| --- | --- | --- |
+| `wp civicrm db clear` | Drop all CiviCRM tables, views, functions and stored procedures from the database. | New |
+| `wp civicrm db cli` | Quickly enter the MySQL command line. | `wp civicrm sql-cli` |
+| `wp civicrm db config` | Show the CiviCRM database connection details. | `wp civicrm sql-conf` |
+| `wp civicrm db connect` | Get a string which connects to the CiviCRM database. | `wp civicrm sql-connect` |
+| `wp civicrm db drop` | Drop the CiviCRM database when it is not shared with WordPress. | New |
+| `wp civicrm db drop-tables` | Drop the CiviCRM tables from the database. | New |
+| `wp civicrm db export` | Export the whole CiviCRM database and print to STDOUT or save to a file.| `wp civicrm sql-dump` |
+| `wp civicrm db functions` | Get the list of CiviCRM functions in the database. | New |
+| `wp civicrm db import` | Loads a whole CiviCRM database. | New |
+| `wp civicrm db is-shared` | Check if CiviCRM shares a database with WordPress. | New |
+| `wp civicrm db procedures` | Get the list of CiviCRM procedures in the database. | New |
+| `wp civicrm db query` | Perform a query on the CiviCRM database. | `wp civicrm sql-query` |
+| `wp civicrm db tables` | Gets a set of CiviCRM tables in the database. | New |
 
-Example: `wp civicrm process-mail-queue -u admin`
+Use `wp help civicrm db <command>` for full details and examples.
 
-#### wp civicrm rest
+### Deprecated Commands
 
-Rest interface for accessing CiviCRM APIs. It can return `xml` or `json` formatted data.
+All previous commands still exist for the time being. However, because they were attached to the top-level `wp civicrm` namespace, it seems sensible to deprecate them in favour of better-namespaced new commands. The following table shows you replacement commands:
 
-#### wp civicrm restore
+| Old Command | New Command |
+| --- | --- |
+| `wp civicrm cache-clear` | `wp civicrm cache flush` |
+| `wp civicrm disable-debug` | `wp civicrm debug disable` |
+| `wp civicrm enable-debug` | `wp civicrm debug enable` |
+| `wp civicrm install` | See [Composite Commands](#composite-commands) below |
+| `wp civicrm member-records` | `wp civicrm job member-records` or `wp civicrm job membership` |
+| `wp civicrm process-mail-queue` | `wp civicrm job process-mail-queue` or `wp civicrm job mailing` |
+| `wp civicrm restore` | Only works with backups made by `wp civicrm upgrade` |
+| `wp civicrm sql-conf` | `wp civicrm db config` or `wp civicrm db conf` |
+| `wp civicrm sql-connect` | `wp civicrm db connect` |
+| `wp civicrm sql-cli` | `wp civicrm db cli` |
+| `wp civicrm sql-dump` | `wp civicrm db export` |
```

Comment:
I'd remove this from 'deprecated' as these command do very different things.   `sql-dump` is the full DB and `db export is just the known CiviCRM tables

Reply:
@kcristiano Good point. However, in the interests of cleaning up the top level of sub-commands, I've kept `wp civicrm sql-dump` as deprecated, un-aliased `wp civicrm db dump` and moved the functionality there in ca4a000d73618560ea6c12f1f9344e0c680752d5

Reply:
FYI the subcommands should eventually become:

```
SYNOPSIS

  wp civicrm <command>

SUBCOMMANDS

  api                     Access the CiviCRM API v3.
  api3                    Access the CiviCRM API v3.
  cache                   Flush the CiviCRM cache.
  core                    Downloads, installs, updates, and manages a CiviCRM installation.
  db                      Utilities for interacting with the CiviCRM database.
  debug                   Enable or disable debugging in CiviCRM.
  job                     Run CiviCRM cron jobs.
  pipe                    Start a Civi::pipe session (JSON-RPC 2.0)
```

There will be `api4` and `ext` coming next, plus any others that people write. I hope you agree that this list is more logical than the previous scattergun approach to the naming scheme of subcommands.